src/pages/commits/ActiveCommits/utils.ts

```diff
@@ -33,3 +34,25 @@ export const groupStatusesByColor = (
 
   return { stats: counts, max, total };
 };
+
+export function findMaxGroupedTaskStats(groupedTaskStats: {
+  [id: string]: GroupedResult;
+}) {
+  const maxes = Object.keys(groupedTaskStats).map(
+    (id) => groupedTaskStats[id].max
+  );
+  return Math.max(...maxes);
+}
+
+export const getAllTaskStatsGroupedByColor = (versions: Version[]) => {
+  const idToGroupedTaskStats: { [id: string]: GroupedResult } = {};
+  versions.forEach((item) => {
+    if (item.version != null) {
+      idToGroupedTaskStats[item.version.id] = groupStatusesByColor(
```

Comment:
I'm sorry i missed this on my last review but we actually already have a `groupStatusesByColor` function. Can we use that instead?  

It would be nice to have this same behavior everywhere on the app since they do the same thing.
https://github.com/evergreen-ci/spruce/blob/73e21154be6da87377ccde8e6e49728b0220d70e/src/pages/patch/buildVariants/utils.ts

Reply:
They are a little different in that:
1. `groupStatusesByColor` also returns `max` and `total` which are used in displaying absolute view and percentage view
2. The map returned by `groupStatusesByColor` is sorted by `sortedStatusColor` so we can ensure that the colored bars inside commit chart are ordered (red first, then green, then ...)